alf/utils/common.py

```diff
@@ -974,7 +974,8 @@ def set_random_seed(seed):
         The seed being used if ``seed`` is None.
     """
     if seed is None:
-        seed = abs(hash(str(os.getpid()) + '|' + str(time.time())))
+        # Has to conver to `int`` otherwise gym will complain
+        seed = int(np.uint32(hash(str(os.getpid()) + '|' + str(time.time()))))
```

Comment:
https://github.com/openai/gym/blob/master/gym/utils/seeding.py#L21

Reply:
newer version of numpy.uint32 will complain if the number is too big

Reply:
> newer version of numpy.uint32 will complain if the number is too big

Now modulo the seed by 2**32